keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/ActionResponse.kt

```diff
@@ -0,0 +1,6 @@
+package com.netflix.spinnaker.keel.api.plugins
+
+data class ActionResponse(
```

Comment:
Maybe rename to `ActionData`?

Reply:
or `ActionDecision`?

Reply:
I like `ActionDecision`, I will switch to that